backend/app/helpers/orders.py

```
@@ -0,0 +1,145 @@
+            "order_quantity": order_item.quantity,
+        }
+        for order_item, product, brand in items
+    ]
+
+
+def get_user_brand_ids(db: Session, user_id: int) -> list:
+    """Get all brand IDs belonging to a user"""
+    user_brands = db.query(models.Brand).filter(models.Brand.user_id == user_id).all()
+    if not user_brands:
+        raise HTTPException(
+            status.HTTP_404_NOT_FOUND, detail="No brands found for this user"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)